src/snapred/backend/dao/request/FarmFreshIngredients.py

```diff
@@ -24,9 +24,6 @@ class FarmFreshIngredients(BaseModel):
     ## needs to be mandatory for diffcal
     cifPath: Optional[str]
 
-    # needs to be mandatory for normalizaiton/reduction
-    smoothingParameter: Optional[float]
-
```

Comment:
This should not be in FarmFreshIngredients, as it is internal data that should be loaded from a record.  The loading from record has been added tot he SousChef method.